postfix/datadog_checks/postfix/postfix.py

```diff
@@ -118,9 +118,11 @@ def _get_config(self, instance):
 
     def _get_postqueue_stats(self, postfix_config_dir, tags):
         pc_output, _, _ = get_subprocess_output(['postconf', 'authorized_mailq_users'], self.log, False)
-        authorized_mailq_users = pc_output.strip('\n').split('=')[1].strip()
 
-        self.log.debug('authorized_mailq_users : %s', authorized_mailq_users)
+        if '=' in pc_output:
```

Comment:
It looks like that some `postconf` fields may not have a value after `=`. Is `authorized_mailq_users` guaranteed to have a nonempty value? It can be possible that `authorized_mailq_users =` will still have an index error.

We could use regex and use groups to check that it is not empty? For example: 

```
>>> import re
>>> pattern = re.compile('(.*) \= ?(.*)')
>>> m = pattern.match('authorized_mailq_users = static:anyone')
>>> m.group(2)
'static:anyone'
>>> 
>>> empty = pattern.match('authorized_mailq_users =')
>>> empty.group(2)
''
```
instead of 
```
if '=' in pc_output:
   ...
```
we can 

```
pattern = re.compile('(.*) \= ?(.*)')
match = pattern.match(pc_output)
if match.group(2):
   ...
```

Reply:
Other option is to check the lenght of `pc_output.strip('\n').split('=')`

Reply:
`postconf` doesn't appear to return any line without an `=` so I guess that would work. will try this

Reply:
Instead of checking for length, I just checked if `pc_output` is not None/empty